UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -335,30 +331,38 @@ private void InitializeEnvironment()
 
             BrainDecideAction += () => { };
             DestroyAction += () => { };
-            AgentSetStatus += (i) => { };
+            AgentSetStatus += i => { };
             AgentResetIfDone += () => { };
             AgentSendState += () => { };
             AgentAct += () => { };
             AgentForceReset += () => { };
 
-
-            // Configure the environment using the configurations provided by
-            // the developer in the Editor.
-            SetIsInference(!m_BrainBatcher.GetIsTraining());
             ConfigureEnvironment();
         }
 
-        private void UpdateResetParameters()
+        static void OnQuitCommandReceived()
         {
-            var newResetParameters = m_BrainBatcher.GetEnvironmentParameters();
-            if (newResetParameters != null)
+#if UNITY_EDITOR
+            EditorApplication.isPlaying = false;
+#endif
+            Application.Quit();
+        }
+
+        void OnRLInputReceived(UnityRLInputParameters inputParams)
+        {
+            m_IsInference = inputParams.isTraining;
```

Comment:
Should this be `!inputParams.isTraining`?

Reply:
Looks like the old Batcher code was `m_IsTraining = firstRlInput.IsTraining`

Reply:
looks like it